config/locales/en/shared.yml

```diff
@@ -313,8 +318,11 @@ en:
         benefits_examples: For example, Universal Credit or Child Benefit.
         pension_examples: Include State, workplace and personal pensions.
         none_selected: None of these
-      identify_types_of_outgoings_form:
-        none_selected: None of these
+        expanded_explanation:
+          heading: Why am I being asked this?
+          list: |
+            Your solicitor will look at your bank statements and put your transactions into the categories you select.
+            We need this information to check if you qualify financially for legal aid.
```

Comment:
you could use the & and <<* notation here to prevent duplicating the content if you think these will always have the same wording.
e.g. on line 186 you can write
`expanded_explanation: &transaction_explain`
and for this section replace lines 321-325 with
```
       expanded_explanation:
          <<: *transaction_explain
```